catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/transformer/impl/EmbeddedListMetacardsHandler.java

```diff
@@ -11,34 +11,36 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.catalog.ui.metacard.workspace.transformations;
+package org.codice.ddf.catalog.ui.metacard.workspace.transformer.impl;
 
 import static java.util.stream.Collectors.toList;
 
 import com.google.common.annotations.VisibleForTesting;
 import ddf.action.ActionRegistry;
 import ddf.catalog.data.Metacard;
+import ddf.catalog.data.MetacardType;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
 import org.codice.ddf.catalog.ui.metacard.workspace.ListMetacardImpl;
-import org.codice.ddf.catalog.ui.metacard.workspace.WorkspaceAttributes;
+import org.codice.ddf.catalog.ui.metacard.workspace.WorkspaceConstants;
+import org.codice.ddf.catalog.ui.metacard.workspace.transformer.EmbeddedMetacardsHandler;
 import org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformer;
 import org.codice.ddf.configuration.SystemBaseUrl;
 
-public class EmbeddedListMetacardsHandler extends EmbeddedMetacardsHandler {
-
+public class EmbeddedListMetacardsHandler implements EmbeddedMetacardsHandler {
```

Comment:
should this be named `Impl`?

Reply:
My understanding is that "`Impl`" is appended when there is only one implementation of the interface expected to be used for most or all cases. This, on the other hand, is one specific implementation of the `EmbeddedMetacardsHandler`; there is also an `EmbeddedQueryMetacardsHandler` and there are likely to be more implementations of `EmbeddedMetacardsHandler`. Let me know if my understanding of that pattern is incorrect and I'll change it.